packages/cli/src/config/loadStoreConfig.ts

```diff
@@ -7,7 +7,7 @@ export async function loadStoreConfig(configPath?: string) {
   const config = await loadConfig(configPath);
 
   try {
-    return parseStoreConfig(config);
+    return parseStoreConfig(config as any);
```

Comment:
why is this needed?

Reply:
`loadConfig` has a return value of `unknown`, and if I don't case it, I get
<img width="832" alt="image" src="https://user-images.githubusercontent.com/508855/225199170-5f08ba14-d024-4455-b55d-5cbc24562aa2.png">

this is happening now because I changed the argument type in `parseStoreConfig` to help with `defineStoreComponents` (to "expand" the mud config into something reasonable/without tons of shorthand and conditional branching)
<img width="542" alt="image" src="https://user-images.githubusercontent.com/508855/225199294-28aa4edb-f20a-40af-a11f-a855f19ed2ba.png">